verifier/verifier.go

```diff
@@ -67,23 +68,28 @@ func (v *verifier) Verify(ctx context.Context, desc ocispec.Descriptor, signatur
 	artifactRef := opts.ArtifactReference
 	envelopeMediaType := opts.SignatureMediaType
 	pluginConfig := opts.PluginConfig
+	logger := log.GetLogger(ctx)
 
+	logger.Debugf("Verify signature against artifact %v referenced as %s", desc.Digest, artifactRef)
```

Comment:
We dont need this line since we would have already emitted
```
Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(`:v1`) because tags are mutable and a tag reference can point to a different artifact than the one signed
Resolved artifact tag `v1` to digest `sha256:cd5eef6b6a6750c9850a8d7b1a5435f35f1a1808d66c74e265f6b7ec290bea47` before signing

```

Reply:
Some library users may use verifier.Verify directly instead of calling notation.go/verify, so those lines will not shown, and the log is helpful.